### PR TITLE
Sdh depmodeltable

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Adding Authentication API/deployment model table
- **Is this PR related to a Monolith release?** No

The wording in the UI that brings the user to this section has been updated to: "Custom integrations that use the Okta Classic Engine API are affected by this setting. Is your app using the Classic Engine API?  **Learn more**"

Screen Shot 2021-11-08 at 11 06 14 AM

### Resolves:

* [OKTA-439081](https://oktainc.atlassian.net/browse/OKTA-439081)

### CODE REVIEW

**Review by brianduffield-okta — COMMENTED** [review]
Line 13, although not part of the update, could use a little massaging, i.e "...that require the customer to sign-in and authenticate," reads better to me.

File: packages/@okta/vuepress-site/docs/concepts/redirect-vs-embedded/index.md [hunk 1]
```diff
@@ -15,7 +15,18 @@ When you develop applications that require the customer to sign-in and then to a
...
 - **Embedded authentication**: A user sign-in flow where the application retains authentication control, without redirection to Okta, using a client-hosted Sign-In Widget, Okta Identity Engine SDKs, or directly with Okta proprietary Identity Engine authentication APIs.
 
-The Identity Engine supports both these deployment and authentication models, and uses authentication protocols to communicate with Identity Server, the entity that acts as the logical location for all identity-related services and resources.
+Identity Engine supports both of these deployment and authentication models, and uses authentication protocols to communicate with Identity Server, which is the entity that acts as the logical location for all identity-related services and resources.
+
+## Deployment models and the Authentication API
+
+The following table details the configurations that define which Authentication API (either Okta Classic Engine or Identity Engine) that your application is using based on your deployment model.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/concepts/redirect-vs-embedded/index.md:22]:
```suggestion
The following table details the configurations that define which Authentication API (either Okta Classic Engine or Identity Engine) your application is using based on your deployment model.
```

File: packages/@okta/vuepress-site/docs/concepts/redirect-vs-embedded/index.md [hunk 2]
```diff
@@ -15,7 +15,18 @@ When you develop applications that require the customer to sign-in and then to a
...
+| ---------------- | --------------------------------- | ---------------------------------- |
+| **Redirect:<br>Okta Hosted Sign-In Widget**<br>To verify that the redirect deployment is used in your application code,<br>check to see if you are using one of these SDKs:<br>[Android](https://github.com/okta/okta-oidc-android), [Angular](https://github.com/okta/okta-angular), [ASP.NET](https://github.com/okta/okta-aspnet), [iOS](https://github.com/okta/okta-oidc-ios), [React](https://github.com/okta/okta-react), [React Native](https://github.com/okta/okta-react-native), [Vue](https://github.com/okta/okta-vue)| The API type depends on your [org configuration.](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) | The API type depends on your [org configuration.](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) |
+| **Embedded:<br>Customer Hosted Sign-In Widget** | Your Sign-In Widget configuration is using:<br>`useInteractionCodeFlow: false`<br>**OR** the `useInteractionCodeFlow` property isn't present | Your Sign-In Widget configuration is using:<br>`useInteractionCodeFlow: true` |
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/concepts/redirect-vs-embedded/index.md:27]: Curious about the all-cap "OR" in bold formatting? Is this necessary?

**Review by brianduffield-okta — APPROVED** [review]
Couple nits but lgtm

File: packages/@okta/vuepress-site/docs/concepts/redirect-vs-embedded/index.md (hunk 2 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/concepts/redirect-vs-embedded/index.md:27]: Just didn't want it to get lost - if you think it's easy to tell there are two situations they could be in without drawing the eye to OR, I'm good with that.

**Pull request merged**